cmd/zed/index/command.go

```diff
@@ -7,24 +7,27 @@ import (
 	"github.com/brimdata/zed/pkg/charm"
 )
 
-var MicroIndex = &charm.Spec{
-	Name:  "microindex",
-	Usage: "microindex <command> [options] [arguments...]",
-	Short: "create and manipulate microindexes",
+var Cmd = &charm.Spec{
+	Name:  "index",
+	Usage: "index <command> [options] [arguments...]",
+	Short: "create and search zed indexes from zed streams or files",
```

Comment:
This reads roughly because "from zed streams or files" applies to "create" but not "search".
```suggestion
	Short: "create and search zed indexes",
```